openseadragon-scalebar.js

```diff
@@ -418,6 +418,21 @@
             return getScalebarSizeAndText(ppmi, minSize, "mi");
         },
         /**
+         * Astronomy units. Choosing the best unit from arcsec, arcminute, and degree
+         */
+        ASTRONOMY: function(ppa, minSize) {
+	    var maxSize = minSize * 2;
```

Comment:
Confused about this one.  Spacing looks fine in my commit (4 spaces in) but this is showing 3...